src/main/java/frc/robot/SystemFactory.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.wpilibj.DutyCycle;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import subSystems.Arm;
+
+public class SystemFactory {
+    public static Arm createArm(){
+        CANSparkMax master = new CANSparkMax(RobotMap.ARM_MASTER, CANSparkMaxLowLevel.MotorType.kBrushed);
+        CANSparkMax follow = new CANSparkMax(RobotMap.ARM_FOLLOW, CANSparkMaxLowLevel.MotorType.kBrushless);
+        DutyCycleEncoder encoder = new DutyCycleEncoder((DutyCycle) RobotMap.ARM_ENCODER);
```

Comment:
You cannot cast `ARM_ENCODER` (which is `HidChannel`) into a `DutyCycle`.

You should use the `DutyCycle(int)` constructor.
```
// channel number is not chosen yet
public static final int ARM_ENCODER = 5;
...
DutyCycleEncoder encoder = new DutyCycleEncoder(RobotMap.ARM_ENCODER);
```